_docs/operations/maintenance.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: guide
+title: StorageOS Docs - Maintenance
+anchor: operations
+module: operations/maintenance
+---
+
+# Maintenance
+
+## Node drain
+
+It is recommended to drain a node before any maintenance operation.
+
+You can either [cordon or drain](/docs/reference/cli/node) a node using the [StorageOS cli](/docs/reference/cli/index). 
+
+A `storageos node cordon` will avoid any new volumes to be hosted in that node. On the other hand, a `storageos node drain` will evict all volumes from the node and will prevent any new volumes to be hosted, as well.
+
+
+####  Hosting primary volumes with replication enabled:
+
+- Existing replicas will be promoted in different nodes
+- The previous primary will become a replica. 
+- The replica is relocated to a different node
+
+#### Hosting primary volumes with replication *not enabled*:
+
+- A replica will be created in a different available node 
+- The new replica will be promoted to primary
+- The old primary volume will be removed
+
+
+#### If there are no available nodes free
+
+There is a variety of reasons for which a volume can not be hosted by a node. For instance, because there are rules in place that define restrictions of location, or because the amount of replicas +1 match the amount of nodes in the cluster.
```

Comment:
"There is a variety" -> "There are a variety"

Reply:
Thank you!
Fix commited: https://github.com/storageos/storageos.github.io/pull/145/commits/8e27e6f1a50283b43209c1ebd1ed6496014a0358